core/templates/pages/library-page/library-page.component.html

```diff
@@ -97,7 +97,7 @@ <h2 [ngClass]="{'active': !libraryWindowIsNarrow && activeGroupIndex === idx}" c
           </div>
 
           <!-- TODO(sll): Add card at end of the row that, when clicked, does showFullResultsPage(group.categories)" -->
-          <div *ngIf="!libraryWindowIsNarrow">
+          <div *ngIf="!libraryWindowIsNarrow" class="carousel-box">
```

Comment:
Please append the class name with `oppia-` and also make sure the class name is unique across the entire codebase.

Reply:
@Lawful2002 yes i have corrected the class name